entries/s/sfu.ca.json

```
@@ -0,0 +1,19 @@
+{
+  "Simon Fraser University": {
+    "domain": "sfu.ca",
```

Looks like https://sfu.ca/ isn't reachable.
In this case it's good to include a url-element:
```suggestion
    "domain": "sfu.ca",
    "url": "https://www.sfu.ca",
```